### DISCUSSION TITLE
Add Option to Use UTC as “Time Zone”

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#533] [Feature Requests]
I have a feature request which is to implement the use of UTC as a "time zone" (actually a time standard if we're being accurate) option when setting the time.

This would be particularly useful for anyone who use the their HERO11 Black(s) e.g. while traveling and don't want to worry about time zone conversions later on.

I am aware of the QR code option for multi-camera shoots; what I am after is a feature that keeps the camera's clock to UTC even when grabbing the time from a connected smartphone regardless of the smartphone's system time.

And if possible when there's a good GPS signal discipline the camera's clock's time to UTC (here I am disregarding the feature which synchronizes the time code track of the video).

**Comment by dnewman-gpsw** [comment]
Can;t you just set via https://gopro.github.io/labs/control/precisiontime_utc/ or set you timezone to GMT.  Use !MSYNC=1 for GPS time sync.

**Reply by AlexLandherr** [reply]
So the "GPS Time Synchronization" Labs feature does not just fix the time code track; it actually sets what I presume is the RTC of the HERO11 Black?

As for the the time zone thing; GMT is not strictly the same as UTC. GMT is a time zone; UTC is a time standard:
[Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time)

What I perhaps failed to communicate adequately was that when I in the camera's settings choose the time zone I want an option that represents actual Coordinated Universal Time (in reality true UTC can only be determined after the fact; here I'm looking at what GPS and my smartphone can provide).
This option would resist jumping to the local time used by my smartphone as soon as it connects via Bluetooth and Wi-Fi.

**Reply by dnewman-gpsw** [reply]
SYNC, GPS set the RTC time, yes. As GPS is UTC (or the GPS equivalent), it has no timezone, the camera will use the previously stored timezone to set the clock.  If you set your TZ to GMT = UTC+0. so close enough, GPS will set UTC time and date on the camera.

**Reply by AlexLandherr** [reply]
Ok, will try that.

I asked because I know some more experienced photographers using big DSLRs always use UTC as their system time to avoid time zone ambiguities.

**Reply by dnewman-gpsw** [reply]
This is no issue if you don't change the timezone.  I don't know if Quik is setting timezone, I so rarely use Quik.

**Reply by AlexLandherr** [reply]
I tried (on the camera without Quick connected) turning of Daylight Savings Time, setting the time zone to "UTC+0" and forcibly setting the time to UTC by manually changing the hour value to the correct one (which for me is local time minus 2 hours since I'm on CEST as of writing this).

And the GPS indicator is white which I understand means it has a satisfactory lock on some satellites.

And since I have GPS Sync enabled in the Labs firmware I should get reasonably good time stamps for the generated videos and images?

**Reply by AlexLandherr** [reply]
A follow-up question:
When I schedule a recording to start; be it time lapse or regular video how to I get rid of the "countdown" that's present even if I have the "timer" option disabled?

E.g. when I set my time lapse or video to start at "12:00" it shows as "12:00:03" when I check the creation time of the file. I've checked but not been able to circumvent it.

**Reply by dnewman-gpsw** [reply]
Can't disable the timer.
For delayed actions what command are using.  Also this should be another thread, and this forum is intend to be helpful for others.

**Reply by AlexLandherr** [reply]
Ok, understood.
